### PR TITLE
new pages in italian

### PR DESCRIPTION
**Pull request by micheleberardi**
i removed the bad quality pages and i added the following pages
/_common-resources/index.md
/community/_index.html
/includes/federation-deprecation-warning-note.md
/includes/index.md
partners/_index.html
_index.html

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by micheleberardi** [comment]
HI
@zacharysarah and @jnardiello
I removed all the old pages and add new ones

**Comment by rlenferink** [comment]
/assign @jnardiello
for review

**Comment by rlenferink** [comment]
ping for review @jnardiello

**Comment by rlenferink** [comment]
Can we make some progress on this ? @jnardiello 
Or can we ping around for some additional Italian reviewers ? @zacharysarah

**Comment by mattiaperi** [comment]
Hi @rlenferink, I had a look to the changes involved in the PR and everything LGTM, we might want to proceed approving it.

**Comment by jnardiello** [comment]
I'm deeply sorry about not giving any feedback! Can you wait a couple days more and I will provide a review?

**Comment by rlenferink** [comment]
Many thanks for reviewing @mattiaperi !

/lgtm
/approve

File: content/it/partners/_index.html [hunk 1]
```diff
@@ -9,10 +9,102 @@
...
+              </h5>La conformità del software assicura che la versione di Kubernetes di ogni fornitore supporti le API richieste.
+              <br><br><br>
+              <button id="conformance" class="button" onClick="updateSrc(this.id)">Guarda i Partner di Conformità</button>
+              <br><br>Interessato a diventare partner <a href="https://www.cncf.io/certification/software-conformance/">Certificati Kubernetes</a>?
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5><b>Partner per la Formazione Kubernetes</b></h5>
+              <br>Professionisti esperti nella formazione con una solida esperienza nella formazione tecnologica nativa in-the-cloud.
```
Comment (jnardiello) [inline content/it/partners/_index.html]: `Professionisti esperti e certificati, con solida esperienza nella formazione tecnologica Cloud Native`

File: content/it/_index.html [hunk 2]
```diff
@@ -1,55 +1,58 @@
 ---
-title: "Container Orchestration a livello di produzione"
-abstract: "Implementazione, ridimensionamento e gestione automatizzata dei container "
+title: "Orchestrazione di Container di Livello Produzione"
```
[the remark this answers is not in the document]
Comment (micheleberardi) [reply content/it/_index.html]: resolved with Orchestrazione Container Production-Grade

File: content/it/partners/_index.html (hunk 1 above)
Comment (micheleberardi) [reply content/it/partners/_index.html]: resolved Professionisti esperti e certificati, con solida esperienza nella formazione tecnologica Cloud Native

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 3]
```diff
@@ -16,16 +16,16 @@ Consulta le guide di [Setup](/docs/setup) per avere degli esempi su come pianifi
...
+ - Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli le distro più adatte alle tue esigenze.
+ - **Se si sta progettando per l'alta disponibilità**, imparare a configurare [cluster in più zone](/docs/concepts/cluster-administration/federation/).
+ - Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes](https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?
+ - Il tuo cluster sarà **on-premises** o **nel cloud (IaaS)**? Kubernetes non supporta direttamente i cluster ibridi. Invece, puoi impostare più cluster.
+ - **Se si sta configurando Kubernetes on-premises**, considerare quale [modello di rete](/docs/concepts/cluster-administration/networking/) si adatta meglio.
+ - Avvierai Kubernetes su **hardware "bare metal"** o su **macchine virtuali (VM)**?
+ - Vuoi **solo eseguire un cluster**, oppure ti aspetti di fare **lo sviluppo attivo del codice del progetto di Kubernetes**? Se la
+   Quest'ultimo, scegliere una distribuzione attivamente sviluppata. Alcune distribuzioni usano solo versioni binarie, ma
```
[the remark this answers is not in the document]
Comment (micheleberardi) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: i would like to review this part with @mattiaperi and @medmedchiheb

[the remark this answers is not in the document]
Comment (jnardiello) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: Ok, let me know if you need any help or feedback.

[the remark this answers is not in the document]
Comment (mattiaperi) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: Please find hereunder my suggestions, keeping a look to the original contents https://kubernetes.io/docs/concepts/cluster-administration/:
- 1st point: `adatta` --> `adatti`. Moreover for coherence with the others PR changes by Jacopo, I would continue using the 2nd person: `Se stai configurando` [...], `considera` [...]
- 2nd point: `Avvierai` --> `Implementerai` or better: `Eseguirai` in coherence with the following point where `run` was translated with `eseguire`, or something that gives more the idea of using and not only starting a new cluster
- 3rd point: in favour of Michele, I would say that the original content is not one of the best examples of clarity, btw I suggest something like: `Vuoi **solo eseguire un cluster**, oppure ti aspetti di fare **lo sviluppo attivo del codice del progetto di Kubernetes**? In quest'ultimo caso, scegli una distribuzione sviluppata attivamente. Alcune distribuzioni utilizzano solo versioni binarie, ma offrono una maggiore varietà di scelte.` But I agree, I'm not even satisfied of my own translation... @micheleberardi and @jnardiello what to think?

[the remark this answers is not in the document]
Comment (jnardiello) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: @mattiaperi 
`1. Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli le distro più adatte alle tue esigenze.` -> So you suggest that this become singular and third person? Something like `Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli la distro che più si adatti alle tue esigenze.`?

2. Agree

3. I know, I find quite confusing the original sentence as well. +1 to your suggested sentence, it's clear and correct.

gogogo

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 4]
```diff
@@ -16,16 +16,16 @@ Consulta le guide di [Setup](/docs/setup) per avere degli esempi su come pianifi
...
- - Il tuo cluster sarà **on-premises** o **nel cloud (IaaS)**? Kubernetes non supporta direttamente i cluster ibridi. Invece, puoi impostare più cluster.
- - **Se si sta configurando Kubernetes on-premises**, considerare quale [modello di rete](/docs/concepts/cluster-administration/networking/) si adatta meglio.
- - Avvierai Kubernetes su **hardware "bare metal"** o su **macchine virtuali (VM)**?
- - Vuoi **solo eseguire un cluster**, oppure ti aspetti di fare **lo sviluppo attivo del codice del progetto di Kubernetes**? Se la
-   Quest'ultimo, scegliere una distribuzione attivamente sviluppata. Alcune distribuzioni usano solo versioni binarie, ma
-   offrire una maggiore varietà di scelte.
- - Familiarizzare con i [componenti](/docs/admin/cluster-components/) necessari per eseguire un cluster.
+ - Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli le distro più adatte alle tue esigenze.
+ - **Se si sta progettando per l'alta disponibilità**, imparare a configurare [cluster in più zone](/docs/concepts/cluster-administration/federation/).
```
Comment (jnardiello) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: `imparare` -> `impara`

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 5]
```diff
@@ -16,16 +16,16 @@ Consulta le guide di [Setup](/docs/setup) per avere degli esempi su come pianifi
...
- - Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes](https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?
- - Il tuo cluster sarà **on-premises** o **nel cloud (IaaS)**? Kubernetes non supporta direttamente i cluster ibridi. Invece, puoi impostare più cluster.
- - **Se si sta configurando Kubernetes on-premises**, considerare quale [modello di rete](/docs/concepts/cluster-administration/networking/) si adatta meglio.
- - Avvierai Kubernetes su **hardware "bare metal"** o su **macchine virtuali (VM)**?
- - Vuoi **solo eseguire un cluster**, oppure ti aspetti di fare **lo sviluppo attivo del codice del progetto di Kubernetes**? Se la
-   Quest'ultimo, scegliere una distribuzione attivamente sviluppata. Alcune distribuzioni usano solo versioni binarie, ma
-   offrire una maggiore varietà di scelte.
- - Familiarizzare con i [componenti](/docs/admin/cluster-components/) necessari per eseguire un cluster.
+ - Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli le distro più adatte alle tue esigenze.
```
[the remark this answers is not in the document]
Comment (jnardiello) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: Sorry, also: not `ad` -> `in alta affidabilita'`

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 6]
```diff
@@ -16,16 +16,16 @@ Consulta le guide di [Setup](/docs/setup) per avere degli esempi su come pianifi
...
+ - **Se si sta progettando per l'alta disponibilità**, imparare a configurare [cluster in più zone](/docs/concepts/cluster-administration/federation/).
+ - Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes](https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?
+ - Il tuo cluster sarà **on-premises** o **nel cloud (IaaS)**? Kubernetes non supporta direttamente i cluster ibridi. Invece, puoi impostare più cluster.
+ - **Se si sta configurando Kubernetes on-premises**, considerare quale [modello di rete](/docs/concepts/cluster-administration/networking/) si adatta meglio.
```
Comment (jnardiello) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: `considerare` -> `considera`

**Comment by xiangpengzhao** [comment]
@micheleberardi please address inline comments and resolve the conflict :)

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (micheleberardi) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: done!!! thanks guys

**Comment by inductor** [comment]
Why does this include changes of many other languages than just Italian?

**Comment by jnardiello** [comment]
I believe it's a merge/rebase gone wrong. Also, as far as I can see, more stuff is being added and committed.  Can we have smaller size pull requests and start merging approved changes? Otherwise it gets next to impossible to review reliably new additions. What do you think?

**Comment by raelga** [comment]
Hello @micheleberardi, do you need any help with the rebase?

Feel free to ping me on Slack at `@rael` :)

**Comment by bucksense** [comment]
hi @rael, I think I lost some comments here, sure I will contact you on slack, Thanks and sorry

**Comment by micheleberardi** [comment]
hi @rael, I think I lost some comments here, sure I will contact you on slack, Thanks and sorry

**Comment by micheleberardi** [comment]
hi @rael i did the rebase let me know

**Comment by inductor** [comment]
@rael @micheleberardi I think the target branch should be the italian translation branch, not to the master.

**Comment by micheleberardi** [comment]
hi @inductor i will contact @rael tomorrow morning(EDT time), apologize for all mess on this

**Comment by raelga** [comment]
Thanks for your time @micheleberardi, the PR now looks great!

/remove-language de
/remove-language en
/remove-language es
/remove-language fr
/remove-language id
/remove-language ja
/remove-language ko
/remove-language pt
/remove-language zh
/remove-area blog

**Comment by micheleberardi** [comment]
@raelga thanks so much for your help

**Comment by inductor** [comment]
Love you guys :)

**Comment by inductor** [comment]
I suppose somebody gotta remove the DNM label and another italian reviewer should give "lgtm" for this to be mergede.

**Comment by rlenferink** [comment]
/approve cancel
/hold cancel

@jnardiello can you verify (maybe give it a quick look) that this is correctly rebased?

**Comment by rlenferink** [comment]
@jnardiello @mattiaperi is there anything which prevents this PR from being merged?

**Comment by inductor** [comment]
you need a '/lgtm' by any Italian reviewers/members @rlenferink @jnardiello @mattiaperi  to get this merged

File: content/it/_index.html [hunk 7]
```diff
@@ -1,62 +1,70 @@
...
-cid: home
+title: Panoramica sull'amministrazione del cluster
+content_template: templates/concept
+weight: 10
 ---
 
-{{< deprecationwarning >}}
+{{% capture overview %}}
+La panoramica dell'amministrazione del cluster è per chiunque crei o gestisca un cluster Kubernetes.
+Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concepts/).
```
Comment (jnardiello) [inline content/it/_index.html]: `Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concepts/).` -> `Presuppone una certa dimestichezza con i [concetti](/docs/concepts/) core Kubernetes.`

File: content/it/_index.html [hunk 8]
```diff
@@ -1,62 +1,70 @@
...
 
-Raggruppa i contenitori che costituiscono un'applicazione in unità logiche per una piu facile gestione. Kubernetes si basa [su 15 anni di esperienza di Google](http://queue.acm.org/detail.cfm?id=2898444) ,combinando idee e messe in pratica suggerite da una comunità.
-{{% /blocks/feature %}}
+Prima di scegliere una guida, ecco alcune considerazioni:
 
-{{% blocks/feature image="scalable" %}}
-#### Planet Scale
+ - Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli la distro che più si adatti alle tue esigenze.
+ - **Se si sta progettando per l'alta disponibilità**, impara a configurare [cluster in più zone](/docs/concepts/cluster-administration/federation/).
+ - Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes](https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?
```
Comment (jnardiello) [inline content/it/_index.html]: `- Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes (https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?`

->

`- Utilizzerai **un cluster di Kubernetes gestito**, come [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) o **sarai tu a gestire il tuo cluster**?`

File: content/it/_index.html [hunk 9]
```diff
@@ -1,62 +1,70 @@
...
+ - Il tuo cluster sarà **on-premises** o **nel cloud (IaaS)**? Kubernetes non supporta direttamente i cluster ibridi. Invece, puoi impostare più cluster.
+ - **Se stai configurando Kubernetes on-premises**, considera quale [modello di rete](/docs/concepts/cluster-administration/networking/) si adatti meglio.
+ - Eseguirai Kubernetes su **hardware "bare metal"** o su **macchine virtuali (VM)**?
+ - Vuoi **solo eseguire un cluster**, oppure ti aspetti di fare **lo sviluppo attivo del codice del progetto di Kubernetes**?
+   In quest'ultimo caso, scegli una distribuzione sviluppata attivamente. Alcune distribuzioni utilizzano solo versioni binarie, ma offrono una maggiore varietà di scelte
+ - Familiarizzare con i [componenti](/docs/admin/cluster-components/) necessari per eseguire un cluster.
 
-Progettato secondo gli stessi principi che consentono a Google di gestire miliardi di container alla settimana, Kubernetes può scalare senza aumentare il tuo team operativo.
+Nota: non tutte le distro vengono mantenute attivamente. Scegli le distro che sono state testate con una versione recente di Kubernetes.
```
Comment (jnardiello) [inline content/it/_index.html]: `Scegli le distro che sono state testate con una versione recente di Kubernetes.` -> `Scegli una distro che sia state testata con una versione recente di Kubernetes.`

File: content/it/_index.html [hunk 10]
```diff
@@ -1,62 +1,70 @@
...
 
-{{% /blocks/feature %}}
+## Managing a cluster
```
Comment (jnardiello) [inline content/it/_index.html]: `## Managing a cluster` - This is still English
-> `## Gestire un cluster`

File: content/it/_index.html [hunk 11]
```diff
@@ -1,62 +1,70 @@
...
-    <br>
-    <br>
-    <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Attend KubeCon in Amsterdam on Mar. 30-Apr. 2, 2020</a>
-</div>
-<div id="videoPlayer">
-    <iframe data-url="https://www.youtube.com/embed/H06qrNmGqyE?autoplay=1" frameborder="0" allowfullscreen></iframe>
-    <button id="closeButton"></button>
-</div>
-{{< /blocks/section >}}
+* [Kubernetes Container Environment](/docs/concepts/containers/container-environment-variables/) descrive l'ambiente per i contenitori gestiti Kubelet su un nodo Kubernetes.
```
Comment (jnardiello) [inline content/it/_index.html]: `contenitori` -> `container`

Are you still using google translate?

File: content/it/_index.html [hunk 12]
```diff
@@ -1,62 +1,70 @@
...
 
-{{< blocks/kubernetes-features >}}
+* [Controllo dell'accesso all'API di Kubernetes](/docs/reference/access-authn-authz/controlling-access/) descrive come impostare le autorizzazioni per gli utenti e gli account di servizio.
```
Comment (jnardiello) [inline content/it/_index.html]: `e gli account di servizio.` -> `ed i Service Account`

File: content/it/_index.html [hunk 13]
```diff
@@ -1,62 +1,70 @@
...
+
+* [Autenticazione](/docs/reference/access-authn-authz/authentication/) spiega l'autenticazione in Kubernetes, incluse le varie opzioni di autenticazione.
```
Comment (jnardiello) [inline content/it/_index.html]: `incluse le varie opzioni di autenticazione` -> `incluse le varie opzioni disponibili`

File: content/it/_index.html [hunk 14]
```diff
@@ -1,62 +1,70 @@
...
+
+* [Autorizzazione](/docs/reference/access-authn-authz/authorization/) è separato dall'autenticazione e controlla come vengono gestite le chiamate HTTP.
+
+* [Utilizzo dei controller di ammissione](/docs/reference/access-authn-authz/admission-controller/) spiega i plug-in che intercettano le richieste al server API Kubernetes dopo l'autenticazione e l'autorizzazione.
```
Comment (jnardiello) [inline content/it/_index.html]: `Utilizzo dei controller di ammissione` -> `Utilizzo degli Admission Controller`

File: content/it/_index.html [hunk 15]
```diff
@@ -1,62 +1,70 @@
...
+### Securing the kubelet
+  * [Master-Node communication](/docs/concepts/architecture/master-node-communication/)
+  * [TLS bootstrapping](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/)
+  * [Kubelet authentication/authorization](/docs/admin/kubelet-authentication-authorization/)
+
+## Optional Cluster Services
+
+* [Integrazione DNS](/docs/concepts/services-networking/dns-pod-service/) descrive come risolvere un nome DNS direttamente su un servizio Kubernetes.
+
+* [Registrazione e monitoraggio delle attività del cluster](/docs/concepts/cluster-administration/logging/) spiega come funziona il logging in Kubernetes e come implementarlo.
```
Comment (jnardiello) [inline content/it/_index.html]: `Registrazione` (in English: `recording`) I think it's out of context here. I think using `Monitoraggio` is enough.

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 16]
```diff
@@ -16,16 +16,15 @@ Consulta le guide di [Setup](/docs/setup) per avere degli esempi su come pianifi
...
- - Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes](https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?
- - Il tuo cluster sarà **on-premises** o **nel cloud (IaaS)**? Kubernetes non supporta direttamente i cluster ibridi. Invece, puoi impostare più cluster.
- - **Se si sta configurando Kubernetes on-premises**, considerare quale [modello di rete](/docs/concepts/cluster-administration/networking/) si adatta meglio.
- - Avvierai Kubernetes su **hardware "bare metal"** o su **macchine virtuali (VM)**?
- - Vuoi **solo eseguire un cluster**, oppure ti aspetti di fare **lo sviluppo attivo del codice del progetto di Kubernetes**? Se la
-   Quest'ultimo, scegliere una distribuzione attivamente sviluppata. Alcune distribuzioni usano solo versioni binarie, ma
-   offrire una maggiore varietà di scelte.
- - Familiarizzare con i [componenti](/docs/admin/cluster-components/) necessari per eseguire un cluster.
+ - Vuoi provare Kubernetes sul tuo computer o vuoi creare un cluster multi-nodo ad alta disponibilità? Scegli la distro che più si adatti alle tue esigenze.
```
Comment (jnardiello) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: `adatti` -> `adatta`

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 17]
```diff
@@ -16,16 +16,15 @@ Consulta le guide di [Setup](/docs/setup) per avere degli esempi su come pianifi
...
+ - **Se si sta progettando per l'alta disponibilità**, impara a configurare [cluster in più zone](/docs/concepts/cluster-administration/federation/).
+ - Utilizzerai **un cluster di Kubernetes ospitato**, come [Motore di Google Kubernetes](https://cloud.google.com/kubernetes-engine/) o **che ospita il tuo cluster**?
```
Comment (jnardiello) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: `ospitato` -> `gestito`

`che ospita il tuo cluster` -> `chi gestirà il tuo cluster?`

File: content/it/partners/_index.html [hunk 18]
```diff
@@ -9,10 +9,101 @@
 <section id="users">
     <main>
         <h5>Kubernetes collabora con i partner per creare una base di codice forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
```
Comment (jnardiello) [inline content/it/partners/_index.html]: This is the literal translation of the English sentence but honestly, it does not sound great in Italian. I personally do not understand what this means (grammatically is correct though). I think we should rephrase this to something else.

File: content/it/partners/_index.html [hunk 19]
```diff
@@ -9,10 +9,101 @@
...
+bigheader: I Partners di Kubernetes
+abstract: La Crescita dell'ecosistema Kubernetes.
+class: gridPage
+cid: partners
+---
+<meta charset="utf-8" />
+
+<section id="users">
+    <main>
+        <h5>Kubernetes lavora in collaborazione con diversi partner al fine di creare un codice dalla base forte e dinamica che supporti uno spettro di piattaforme complementari.</h5>
```
Comment (jnardiello) [inline content/it/partners/_index.html]: Same as before
> This is the literal translation of the English sentence but honestly, it does not sound great in Italian. I personally do not understand what this means (grammatically is correct though). I think we should rephrase this to something else.

File: content/it/partners/_index.html [hunk 20]
```diff
@@ -9,10 +9,101 @@
...
         <div class="col-container">
           <div class="col-nav">
             <center>
               <h5>
+                <b>Fornitori di Servizi Certificati Kubernetes</b>
+              </h5>
+              <br>Fornitori di servizi controllati con grande esperienza nell'aiutare le imprese ad adottare con successo Kubernetes.
+              <br><br><br>
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">Guarda i Partners di KCSP</button>
```
Comment (jnardiello) [inline content/it/partners/_index.html]: `Guarda i Partners di KCSP` -> `Guarda i Partners KCSP`

File: content/it/partners/_index.html [hunk 21]
```diff
@@ -9,10 +9,101 @@
...
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>Distribuzioni Kubernetes Certificate, Piattaforme Hosted e Installers</b>
+              </h5>La conformità del software assicura che la versione di Kubernetes di ogni fornitore supporti le API richieste.
+              <br><br><br>
+              <button id="conformance" class="button" onClick="updateSrc(this.id)">Guarda i Partner di Conformità</button>
+              <br><br>Interessato a diventare partner <a href="https://www.cncf.io/certification/software-conformance/">Certificati Kubernetes</a>?
```
Comment (jnardiello) [inline content/it/partners/_index.html]: `Certificati` -> `Certificato`

File: content/it/_index.html (hunk 11 above)
Comment (micheleberardi) [reply content/it/_index.html]: Nope, maybe I forgot from the previous version. sorry about that

File: content/it/partners/_index.html (hunk 19 above)
Comment (micheleberardi) [reply content/it/partners/_index.html]: Hi @mattiaperi, any idea how we can rephrase this sentence?
Thanks in advance for your help

File: content/it/partners/_index.html (hunk 18 above)
Comment (micheleberardi) [reply content/it/partners/_index.html]: Hi @mattiaperi, any idea how we can rephrase this sentence?
Thanks in advance for your help

File: content/it/partners/_index.html (hunk 19 above)
Comment (jnardiello) [reply content/it/partners/_index.html]: Much better overall, I suggest this:

```
Kubernetes collabora con vari partner per creare un sorgente di codice robusto e al tempo stesso dinamico, in grado di supportare una moltitudine di piattaforme.
```
-> 
```
Kubernetes collabora con vari partner per creare codice sorgente che sia robusto e al tempo stesso dinamico, in grado di supportare una moltitudine di piattaforme.
```

**Comment by micheleberardi** [comment]
I fixed all the issues. Thanks to @janrdiello and @mattiaperi for help and support

**Comment by jnardiello** [comment]
LGTM, great job @micheleberardi

File: content/it/docs/concepts/overview/what-is-kubernetes.md [hunk 22]
```diff
@@ -1,157 +1,92 @@
 ---
-title: Che cos'è Kubernetes? 
+reviewers:
+- bgrant0607
+- mikedanese
```
Comment (rlenferink) [inline content/it/docs/concepts/overview/what-is-kubernetes.md]: One tiny nit: can you remove this block?

Comment (micheleberardi) [reply content/it/docs/concepts/overview/what-is-kubernetes.md]: Hi @rlenferink  done 
thanks for the support
